README.md

```diff
@@ -46,7 +48,8 @@ kubescape scan --submit --enable-host-scan --verbose
 
 </br>
 
-### Click [👍](https://github.com/kubescape/kubescape/stargazers) if you want us to continue to develop and improve Kubescape 😀
+### Please Give a [⭐](https://github.com/kubescape/kubescape/stargazers) the repo inorder to show support 
+### and if you want us to continue to develop and improve Kubescape 😀 .
```

Comment:
issue: the heading splits in two, sentence should be grammatically improved

Let’s change it so the header stays put and make the sentence a bit more idiomatic.

```suggestion
### [Star ⭐](https://github.com/kubescape/kubescape/stargazers) the repo if you want us to continue developing and improving Kubescape 😀
```